### ISSUE TITLE
Make a jQuery independent version as well?

### ISSUE THREAD

**Issue by ShashankaNataraj** [FezVrasta/snackbarjs#26]
Hi,

Snackbar is quite really a very small program. Why do we have to depend on jQuery? Is it possible to make a non jQuery dependent version? I would be interested in making contributions!

**Comment by FezVrasta** [comment]
Yes it could be nice, I never worked on it because almost every project out there uses jquery..

**Comment by mbrevda** [comment]
Or a React version!
